### PR TITLE
Health check endpoints

### PR DESCRIPTION
**Pull request by BenBJSS**

### CODE REVIEW

File: app/test-utils/testFunctions/testHealthStatus.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import request from 'supertest';
+import { ApiProvider } from '../../apiProvider';
+import { App } from '../../../app';
+import routes from '../../pages/health/routes';
+
+export const testHealthStatus = (route, { BuyingCatalogueApi, DocumentApi, expected }) => () => {
```
Comment (kat1906) [inline app/test-utils/testFunctions/testHealthStatus.js]: please change as detailed in MP

**Comment by kat1906** [comment]
Please can you prefix all branches and pull requests for all repos titles with the story number from devops

**Comment by alibombaye** [comment]
> Please can you prefix all branches and pull requests for all repos titles with the story number from devops

Good shout Katie.

File: .eslintrc.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ module.exports = {
     "array-callback-return": "off",
     "linebreak-style": "off",
     "no-console": 1,
-    "import/no-extraneous-dependencies": ["error", { "devDependencies": ['**/*test.js', '**/testcafeRunner.js'] }]
+    "import/no-extraneous-dependencies": ["error", { "devDependencies": ["**/test-utils/**", '**/*test.js', '**/testcafeRunner.js'] }]
```
Comment (alibombaye) [inline .eslintrc.js]: do we still need this?

Comment (BenBJSS) [reply .eslintrc.js]: Thought as I added it in to match MP i might as well keep them consistent

**Pull request merged**